lib/src/screens/new_wallet/new_wallet_type_page.dart

```diff
@@ -67,8 +58,6 @@ class WalletTypeFormState extends State<WalletTypeForm> {
 
   WalletType? selected;
   List<WalletType> types;
-  // FIX-ME: Replace Flushbar
-  // Flushbar<void>? _progressBar;
```

Comment:
I took a look at the history of this class and found that the usage for the `_progressBar` was moved to another class, so I removed it since it's no longer needed here